networks.json

```diff
@@ -7,8 +7,8 @@
     },
     "4": {
       "links": {},
-      "address": "0x1b94BF378B910e5be70b59e774867a1c919821C7",
-      "transactionHash": "0xf8256a6b6c92a95f9342d0dbb8afee83d1dd6bec8510278893f568ae0114ee3f"
+      "address": "0x9ebF59B1F7F58Db871189a17c078B456014fe515",
+      "transactionHash": "0x992d706340cae916c667cb0733b8dc4e80c66afbde56a5d81ee6af46ba233838"
```

Comment:
Note, I will also have to redeploy mainnet if this change is accepted.

Reply:
I thought the changes to `fleetFactory` were simply just lint fixes? Did you change something else that meant we had to redeploy?

Reply:
I actually made it return the fleet address on top of emitting them as an event. This way we can "call" the method in the dry run mode.

Reply:
Noice! Smort!